tests/many_to_many/tests.py

```diff
@@ -454,3 +454,29 @@ def test_clear(self):
         self.assertQuerysetEqual(self.a4.publications.all(), [])
         self.assertQuerysetEqual(self.p2.article_set.all(),
                                  ['<Article: NASA finds intelligent life on Earth>'])
+
+    def test_inherited_models_selects(self):
+        """
+        #24156 - Need to test that objects from each of child models
```

Comment:
Objects from child models where the parent's m2m field uses `related_name='+'` should be retrieved correctly.